funcs.sh

```diff
@@ -98,49 +98,20 @@ function runreq
 }
 function gpureq
 {
-  if [[ -f "/proc/driver/nvidia/version" ]]; then
```

Comment:
From what I can tell, this first if else is all about checking to see if the correct driver versions are installed. That doesn't really matter with the toolkit... so I removed it...